importer/api/__init__.py

```diff
@@ -144,6 +142,16 @@ def import_children(course, element, parent):
         content_xml=etree.tostring(element),
         mpath=mpath,
     )
+    if element.tag == "video":
```

Comment:
Since we expect to get quite a few of these parsers on different elements, maybe move this out to a function.  It might be cool to just have a dictionary of functions to do parsing based on the tag